c10/core/Device.cpp

```diff
@@ -23,6 +23,7 @@ DeviceType parse_type(const std::string& device_string) {
       {"hip", DeviceType::HIP},
       {"msnpu", DeviceType::MSNPU},
       {"xla", DeviceType::XLA},
+      {"quantized", DeviceType::QUANTIZED},
```

Comment:
quantized is not a device -- there should be some other way to do this IMO.

Reply:
yes, this is still under discussion, I'll remove